diff --git a/dist/plugins/postcss/index.js b/dist/plugins/postcss/index.js
index a59ad00ba09e4654c38c306a81044c38f96d4ed6..444ea15b66cdd30f023b2805bdaf9687b91bfc07 100644
--- a/dist/plugins/postcss/index.js
+++ b/dist/plugins/postcss/index.js
@@ -20,7 +20,8 @@ const resolveConfig = config => {
         })
         : [];
     const inputs = plugins.map(toDeferResolve);
-    return plugins.includes('tailwindcss') ? [...inputs, toDependency('postcss')] : inputs;
+    // can be deleted after https://github.com/webpro-nl/knip/pull/1069 is released
+    return ['tailwindcss', '@tailwindcss/postcss'].some(tailwindPlugin => plugins.includes(tailwindPlugin)) ? [...inputs, toDependency('postcss')] : inputs;
 };
 export default {
     title,
